test/parallel/test-fs-stat.js

```diff
@@ -102,23 +102,33 @@ fs.stat(__filename, common.mustCall(function(err, s) {
   assert.ok(s.mtime instanceof Date);
   assert.ok(s.ctime instanceof Date);
   assert.ok(s.birthtime instanceof Date);
+  assert.strictEqual(typeof s.atimeMs, 'number');
+  assert.strictEqual(typeof s.mtimeMs, 'number');
+  assert.strictEqual(typeof s.ctimeMs, 'number');
+  assert.strictEqual(typeof s.birthtimeMs, 'number');
 }));
 
 fs.stat(__filename, common.mustCall(function(err, s) {
-  const json = JSON.parse(JSON.stringify(s));
+  const parsed = JSON.parse(JSON.stringify(s));
   const keys = [
     'dev', 'mode', 'nlink', 'uid',
-    'gid', 'rdev', 'ino',
-    'size', 'atime', 'mtime',
-    'ctime', 'birthtime'
+    'gid', 'rdev', 'ino', 'size',
+    'atime', 'mtime', 'ctime', 'birthtime',
+    'atimeMs', 'mtimeMs', 'ctimeMs', 'birthtimeMs'
   ];
   if (!common.isWindows) {
     keys.push('blocks', 'blksize');
   }
   keys.forEach(function(k) {
     assert.ok(
-      json[k] !== undefined && json[k] !== null,
+      parsed[k] !== undefined && parsed[k] !== null,
       k + ' should not be null or undefined'
     );
   });
+  
+  const newFields = ['atimeMs', 'mtimeMs', 'ctimeMs', 'birthtimeMs'];
+  newFields.forEach(function(k) {
+    assert.strictEqual(typeof s.birthtimeMs, 'number', `${k} should a number`);
```

Comment:
Ack.
But we both missed the bug: `typeof s.birthtimeMs` was not parameterized 🤦‍♂️